pymath/lib/math.py

```
@@ -0,0 +1,8 @@
+def is_prime(n):
+  """Checks if a number is a prime number."""
+  if n <= 1:
+    return False
+  for i in range(2, int(n**0.5) + 1):
+    if n % i == 0:
+      return False
+  return True
```

Ruff (0.12.2)

Use `return all(n % i != 0 for i in range(2, int(n ** 0.5) + 1))` instead of `for` loop

Replace with `return all(n % i != 0 for i in range(2, int(n ** 0.5) + 1))`

(SIM110)